src/dom/DomUtil.js

```diff
@@ -33,6 +33,8 @@ export var TRANSITION = testProp(
 export var TRANSITION_END =
 	TRANSITION === 'webkitTransition' || TRANSITION === 'OTransition' ? TRANSITION + 'End' : 'transitionend';
 
+export var DEG_TO_RAD = Math.PI / 180;
+export var RAD_TO_DEG = 180 / Math.PI;

```

Comment:
These exports used to be removed, so do not reintroduce them again.

Reply:
It is based on the [current version](https://github.com/Leaflet/Leaflet/blob/ce266a8c77731f83e8e9e674cf20712012f9ab19/src/dom/DomUtil.js#L234)

Reply:
Read again.
Those exports initially was part of 0.x api.
But since 1.x we do not have them.

You are free to set up corresponding constants locally, without exposing them in api.

Reply:
It is done